test_discovery.py

```diff
@@ -548,8 +538,7 @@ def main(discovery_directory, test_file, shutdown=True):
     except Exception:
         logger.exception("Error: Check test file for formatting errors", exc_info=True)
 
-    if shutdown:
-        shutdown_discovery()
+    shutdown_discovery(shutdown)
         
     if not success:
         exit(1)
```

Comment:
The rest of the exits in this script are `sys.exit(1)`. Consistency is nice.
```suggestion
         sys.exit(1)
```